src/Groups/directproducts.jl

```diff
@@ -410,13 +565,43 @@ end
     normal_subgroup(W::WreathProductGroup)
 
 Return `G`, where `W` is the wreath product of `G` and `H`.
+
+# Examples
+```jldoctest
+julia> G =cyclic_group(3)
+<pc group of size 3 with 1 generators>
+
+julia> H = symmetric_group(2)
+Sym( [ 1 .. 2 ] )
+
+julia> W = wreath_product(G,H)
+<group of size 18 with 2 generators>
+
+julia> normal_subgroup(W)
+<pc group of size 3 with 1 generators>
+```
 """
 normal_subgroup(W::WreathProductGroup) = W.G
 
 """
     acting_subgroup(W::WreathProductGroup)
 
 Return `H`, where `W` is the wreath product of `G` and `H`.
+
+# Examples
+```jldoctest
+julia> G =cyclic_group(3)
```

Comment:
```suggestion
julia> G = cyclic_group(3)
```
Similar in other places

Reply:
You adjusted this in a few places, but there are still plenty of places where there is no space after `=`